src/subscription_manager/gui/data/glade/factsdialog.glade

```diff
@@ -45,6 +45,8 @@
                     <property name="visible">True</property>
                     <property name="can_focus">False</property>
                     <property name="xalign">0</property>
+                    <property name="wrap">True</property>
+                    <property name="wrap_mode">char</property>
                     <property name="selectable">True</property>
```

Comment:
It looks quite good. I'm using wrap_mode: char, because UUID will be the last on the line.